tests/functional/scenario1/DatadirScenarioTest.php

```diff
@@ -209,13 +230,23 @@ public function testUsersHaveCorrectAccessAfterProvisioning(): void
         $masterConnection->query('DROP TABLE IF EXISTS read_schema_table');
         $masterConnection->query('CREATE TABLE read_schema_table (id INT)');
         $masterConnection->query('INSERT INTO read_schema_table VALUES (9), (8), (7)');
-        // need to re-grant to make new table visible
-        $readOnlyRole = $this->namingConventions->getRoRoleFromSchemaName($readSchema);
-        $masterConnection->grantSelectOnAllTablesInSchemaToRole($readSchema, $readOnlyRole);
 
-        // disconnect
         unset($masterConnection);
 
+        $user1connection = $this->getConnectionForUserFromUserConfig($user1ConfigArray);
+        try {
+            $user1connection->fetchAll('SELECT * FROM read_schema_table');
```

Comment:
Tuhle tabulku bych jen přejmenoval, ať je jasný, o co jde - třeba `new_read_schema_table`, `read_schema_table_with_deferred_grant` nebo něco podobnýho. Chvilku jsem tápal co se na testu mění a proč, v popisku nic nebylo.

Reply:
Jenže oba dva tyhle názvy dávají smysl jen na tomhle místě. Pak už není "new" ani "with deferred grant". Tohle prostě jen dokumentuje chování, že dokud se nepustí config row znovu, tak se to nedá číst.